addon/mixins/ajax-request.js

```diff
@@ -615,7 +616,7 @@ export default Mixin.create({
    */
   generateDetailedMessage(status, headers, payload, requestData) {
     let shortenedPayload;
-    const payloadContentType = headers['Content-Type'] || 'Empty Content-Type';
+    const payloadContentType = getHeader(headers, 'Content-Type') || 'Empty Content-Type';
```

Comment:
Is there some default `content-type` value that is assumed and should be used? Or is `Empty Content-Type` appropriate?